src/opers.c

```diff
@@ -2180,6 +2148,8 @@ static ALWAYS_INLINE Obj DoOperationNArgs(Obj  oper,
             CacheMissStatistics[(prec >= CACHE_SIZE) ? CACHE_SIZE : prec]
                                [n]++;
         }
+        if (prec > 0)
```

Comment:
If this fixes a bug should it have its own ticket and a regression test?

Reply:
Looks good. Just FYI in my WIP change (C_)ELM_FLAGS  is automatically safe, but that's easy to deal with. LEN_FLAGS also goes.

Reply:
The second of there is really a general comment. I put it in the wrong place.

Reply:
True, `OperationNext` fix should go to a separate PR. Though I am not completely sure how to "regression test" it, as it only is used when `COUNT_OPERS` is `#defined`... I guess we'd have to set up another GAP build which enables it? But there are also several other #defines we never test right now, such as `CHECK_NDENSE_BAGS`, `DEBUG_MASTERPOINTERS`, `DEBUG_GASMAN_MARKING`, `DEBUG_RATIONALS`